libui-napi.d.ts

```diff
@@ -273,20 +342,85 @@ export abstract class UiControl {
 	/**
 	 * Set or return whether the control is enabled.
 	 */
-	enabled: boolean;
+	public enabled: boolean;
 	/**
 	 * Set or return whether the control is visible.
 	 */
-	visible: boolean;
+	public visible: boolean;
 	/**
 	 * Set or return whether the control is a top level one.
 	 */
-	toplevel: boolean;
+	public toplevel: boolean;
 	/**
 	 * Create a new UiControl object.
 	 */
 	constructor(handle: any);
```

Comment:
```suggestion
```

This is the last remaining "private/internal" constructor

Reply:
Why you said constructor? Anyway, I never exactly understand how that could be used, apart for read. Do you never try to set toplevel=false on a toplevel control?

Reply:
No 😄 , I meant this part:

https://github.com/parro-it/libui-napi/blob/dade525277d34f42de9841075701e70b6cd3f614/libui-napi.d.ts#L354-L357

Reply:
Ah, sure! I guess it should be `protected`?

Reply:
I would just remove it, because libui-napi (`js/*.js`) itself isn't typechecked. This is just the public API.

Reply:
And toplevel is just a getter, so it should be readonly:

https://github.com/parro-it/libui-napi/blob/5e32314516a163bc91da36cb709382cbc656e56f/js/ui-control.js#L36-L42